composer/workflows/create_dataform_workflow_invocation.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START composer_dataform_workflow_invocation]
+"""
+Example Airflow DAG that creates a Dataform compilation result and starts a Dataform workflow invocation. This Airflow DAG uses Google Dataform Airflow operators. For more information about Google Dataform Airflow operators, see https://airflow.apache.org/docs/apache-airflow-providers-google/stable/operators/cloud/dataform.html?highlight=dataform#google-dataform-operators
+"""
```

Comment:
This will not render well in the sample pages with such long horizontal scroll bar :/
```suggestion
"""
Example Airflow DAG that creates a Dataform compilation result and starts a Dataform workflow invocation.
This Airflow DAG uses Google Dataform Airflow operators. For more information about Google Dataform Airflow operators,
see https://airflow.apache.org/docs/apache-airflow-providers-google/stable/operators/cloud/dataform.html?highlight=dataform#google-dataform-operators
"""
```